README.md

```diff
@@ -451,11 +451,11 @@ Delete *addrs* from the KB memory and returns boolean status.
 
 ```python
 from sc_client.client import generate_elements, set_link_contents
-from sc_client.constants import sc_types
+from sc_client.constants import sc_type
 from sc_client.models import ScConstruction, ScLinkContent, ScLinkContentType
 
 construction = ScConstruction()  # Create link for example
-construction.generate_link(sc_types.LINK_CONST, ScLinkContent("One", ScLinkContentType.STRING))
+construction.generate_link(sc_type.CONST_NODE_LINK, ScLinkContent("One", ScLinkContentType.STRING))
 link = generate_elements(construction)[0]
 
 link_content = ScLinkContent("Two", ScLinkContentType.STRING, link)
```

Comment:
'Erase elements' section does not contain any erase_elements examples

Reply:
It has nothing to do with this PR

Reply:
Okay